internal/utils/temp_test.go

```
@@ -37,60 +34,50 @@ func TestCleanupTempDir(t *testing.T) {
 	if err := os.MkdirAll(recentDir, 0755); err != nil {
 		t.Fatalf("Failed to create recent test directory: %v", err)
 	}
 
-	// Run cleanup
 	CleanupOldTempDirs()
 
-	// Check that old directory was cleaned
 	if _, err := os.Stat(oldDir); !os.IsNotExist(err) {
 		t.Errorf("Old directory should be cleaned up: %s", oldDir)
-		os.RemoveAll(oldDir) // cleanup in case test fails
+		os.RemoveAll(oldDir)
```

golangci-lint (2.12.2)

[error] Error return value of `os.RemoveAll` is not checked

(errcheck)